src/LaplaceApproximationModule.jl

```diff
@@ -1,3 +1,21 @@
+module LaplaceApproximationModule
+
+using ..API
+
+export LaplaceApproximation
+export build_laplace_objective, build_laplace_objective!
+
+using ForwardDiff: ForwardDiff
+using Distributions
+using LinearAlgebra
+using Statistics
+using StatsBase
+
+using ChainRulesCore: ignore_derivatives, NoTangent, @thunk
+using ChainRulesCore: ChainRulesCore
+
+using AbstractGPs: LatentFiniteGP, ApproxPosteriorGP
```

Comment:
I also really like how this makes clear exactly what modules the code in this approach depends on

Reply:
Do submodules inherit the `using` imports of the parent module? Because if they do, then `Distributions` would already be imported in `utils.jl` no?